bittensor/utils/__init__.py

```diff
@@ -370,3 +376,32 @@ def validate_chain_endpoint(endpoint_url: str) -> tuple[bool, str]:
     if not parsed.netloc:
         return False, "Invalid URL passed as the endpoint"
     return True, ""
+
+
+def unlock_key(wallet: "Wallet", unlock_type="cold") -> "UnlockStatus":
+    """
+    Attempts to decrypt a wallet's coldkey or hotkey
+    Args:
+        wallet: a Wallet object
+        unlock_type: the key type, 'cold', 'hot', or 'coldkeypub'
+    Returns: UnlockStatus for success status of unlock, with error message if unsuccessful
+    """
+    if unlock_type == "cold":
+        unlocker = "unlock_coldkey"
+    elif unlock_type == "hot":
+        unlocker = "unlock_hotkey"
```

Comment:
pls replace `cold` with `coldkey` and 'hot` with `hotkey`. also across the code where sdk use this.